modules/support-collecting-host-network-trace.adoc

```diff
@@ -72,3 +107,25 @@ tmp/captures
 +
 <1> The packet captures are stored in directories that identify the hostname, container, and file name.
 If you did not specify the `--node-selector` argument, then the directory level for the hostname is not present.
+endif::openshift-origin[]
+
```

Comment:
@peer-review-squad Existing text in the following edit. I updated only the image name in the code block and added ifdef